.github/workflows/testing.yml

```diff
@@ -184,7 +184,7 @@ jobs:
           rm -f ${ARCHIVE_NAME}
 
       - name: Setup golang for the connector and tests
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
```

Comment:
It is better to bump versions in a separate commit.

Reply:
changed